Dockerfile-python-build

```diff
@@ -0,0 +1,43 @@
+ARG BASE_IMAGE=ubuntu:18.04
+FROM --platform=$TARGETPLATFORM $BASE_IMAGE
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LANG=C.UTF-8
+
+RUN set -ex \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        build-essential \
+        libbz2-dev \
+        libffi-dev \
+        libgdbm-dev \
+        libncurses5-dev \
+        libnss3-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        libssl-dev \
+        wget \
+        xz-utils \
+        zlib1g-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG PYTHON_INSTALL_PREFIX=/src/python
```

Comment:
I would propose to remove default value and add checking if it is empty. Like so
```
ARG PYTHON_INSTALL_PREFIX
RUN test -n "$PYTHON_INSTALL_PREFIX" || (echo "PYTHON_INSTALL_PREFIX must be provided" && false)
```